Source/Core/Core/HW/WiimoteEmu/Attachment/Guitar.cpp

```diff
@@ -83,8 +99,17 @@ void Guitar::GetState(u8* const data)
     gdata->sy = static_cast<u8>((y * 0x1F) + 0x20);
   }
 
-  // TODO: touch bar, probably not
-  gdata->tb = 0x0F;  // not touched
+  // slider bar
+  if (m_slider_bar->controls[0]->control_ref->BoundCount())
+  {  // if user has not mapped controls for slider bar, tell game it's untouched
```

Comment:
This comment would probably be clearer if you put it above the line `gdata->sb = 0x0F;`